src/library.js

```diff
@@ -1875,6 +1875,7 @@ LibraryManager.library = {
 
     return { family: family, addr: addr, port: port };
   },
+  $writeSockaddr__docs: '/** @param {number=} addrlen */',
```

Comment:
why are things like this needed? (what warning happens without it?)

Reply:
closure treats all argument as required unless explicitly marked as optional.  Here its the `=` char that marks the argument as optional. 

As an alternative we could use the ES6 syntax in the actual function signature `, addrlen = 0`...but that adds some extra bytes to the output JS (maybe closure can remove this so its ok?).  For now this solution is what we've been going with.

Reply:
I see, thanks. sgtm as is.